unit_tests/test_xunit.py

```diff
@@ -101,6 +113,39 @@ class DummyStream:
         f.close()
         return data
 
+class TestXMLOutputWithXMLAndPrefix(BaseTestXMLOutputWithXML):
+    def setUp(self):
+        super(TestXMLOutputWithXMLAndPrefix, self).setUp()
+        self.configure([
+            "--with-xunit",
+            "--xunit-file=%s" % self.xmlfile,
+            "--xunit-prefix-with-testsuite-name"
```

Comment:
I thought adding the default would be enough. Additional test is in 0214d2e66d0ef69dc36d3703f377ff8d0fc99f08